sample/src/debug/java/com/polidea/cockpit/sample/util/Extensions.kt

```diff
@@ -9,6 +10,7 @@ internal fun MainActivity.initViews() {
     findViewById<View>(R.id.edit_values_button).setOnClickListener {
```

Comment:
This is not a proper extension. What about creating BaseMainActivity in both variants and implement initView() method in their bodies? findViewById call won't be necessary.